src/index.ts

```diff
@@ -55,7 +55,11 @@ async function axiosFetch (
   try {
     result = await axios.request(config);
   } catch (err) {
-    result = err.response;
+    if (err.response) {
```

Comment:
Without this change the original error is masked by a new error: `TypeError: Cannot read property 'headers' of undefined.`. This is because the code below assumes that `response` is defined and attempts to read attributes from it.